ci/launch_cluster.sh

```diff
@@ -38,16 +38,24 @@ echo "Using: ${TEMPLATE}"
 envsubst <<EOF > "$CONFIG_PATH"
 ---
 launch_config_version: 1
-template_url: $TEMPLATE
```

Comment:
Where did this go? We use two specific DC/OS pull requests:
* https://github.com/dcos/dcos/pull/1739 for DC/OS with latest Marathon
* https://github.com/mesosphere/dcos-enterprise/pull/1296 for DC/OS EE with latest Marathon